brave/brave-core
# NTP background custom image

Resolves https://github.com/brave/brave-browser/issues/19640

`brave_new_tab_page.mojom` is introduced for communicating NTP and browser.
Currently it has only custom background related interfaces.
When user chooses custom background image, `BraveNewTabPageHandler` sanitizes that image
and save. After that, that image is delivered to NTP always. 

![image](https://user-images.githubusercontent.com/6786187/147903170-9edf5497-bb1f-4186-87f8-f56a82a76c20.png)

## Submitter Checklist:


## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:
`npm run test brave_unit_tests -- --filter=*ViewCounter*`

1. Launch browser with clean profile and check brave background is displayed
2. Choose custom background or brave brave background via NTP customize dashboard
